azure-functions/acbs-function/api.js

```diff
@@ -195,8 +195,8 @@ const createFacilityCovenantId = (payload) => postMdm('numbers', payload);
 // *** TFS ***
 
 // GET
-const getFacility = (facilityId) => get(`facilities/${facilityId}`);
-const getLoanId = (facilityId) => get(`facilities/${facilityId}/loans`);
+const getFacility = (facilityIdentifier) => get(`facilities/${facilityIdentifier}`);
```

Comment:
Can we add some documentation here with examples of some facility identifiers? Something like 

```ts
@param {String} facilityIdentifier - Facility ACBS ID - "Bond" = 1, "Cash" = 2
```

Reply:
Have added to all the functions.